unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -165,6 +165,38 @@ public UTF8String toLowerCase() {
     return UTF8String.fromString(toString().toLowerCase());
   }
 
+  public UTF8String initCap(final byte[] byteArr) {
+    String isoString = null;
+    try {
+      isoString = new String(byteArr, "US-ASCII");
```

Comment:
tarekauel ,
The Greece characters ε don't have the Uppercase in their usage.

http://baike.baidu.com/link?url=nKnZXnKJYQwbzKeL8BOU8HegSt-y48S0YphsOpc6hU4IisPoXqUHuevcuzcvGPpWutC2TDxLkTiLTlUuYQvwsa

Skip the Chinese explanation and check the bottom.